Formula/ola.rb

```diff
@@ -1,48 +1,35 @@
 class Ola < Formula
   desc "Open Lighting Architecture for lighting control information"
   homepage "https://www.openlighting.org/ola/"
-  url "https://github.com/OpenLightingProject/ola/releases/download/0.10.2/ola-0.10.2.tar.gz"
```

Comment:
The release is here now:
https://github.com/OpenLightingProject/ola/releases/download/0.10.3/ola-0.10.3.tar.gz